SabreTools.DatTools/Rebuilder.cs

```diff
@@ -27,6 +30,7 @@ public class Rebuilder
         /// Logging object
         /// </summary>
         private static readonly Logger logger = new Logger();
+        private static string symlinkDirReport = "";
```

Comment:
You need to add a space between this and the previous variable. You also need to add a summary explaining what this is.

Reply:
Please use `string.Empty` instead of `""`.